desired_distance: 1.0
alpha: 1.2
k_fi: 0.3
k_d: 2
